src/main/scala/com/fulcrumgenomics/util/GeneAnnotations.scala

```diff
@@ -114,4 +115,84 @@ object GeneAnnotations {
     require(start <= end, "start is greater than end when creating an exon")
     def length: Int = end - start + 1
   }
+
+  /** Trait that gene biotypes will extends */
+  sealed trait GeneBiotype extends EnumEntry {
+    /** The biotype name in the GFF */
+    def key: String
+    /** True if this biotype represents a protein coding feature */
+    def isCoding: Boolean
+  }
+
+  /** Enum to represent gene biotypes in a GFF */
+  object GeneBiotype extends FgBioEnum[GeneBiotype] {
```

Comment:
1. Is there a URL you can link to that describes the set of possible biotypes in a GFF from NCBI's RefSeq?  
2. Here are the [biotypes from GENCODE and Ensembl](https://www.gencodegenes.org/pages/biotypes.html).  I'd put a note in that we only support biotypes from (1), but future biotypes could be added.

I am concerned this is too NCBI-specific now, any thoughts on how to make that clearer?

Reply:
I'm not sure what to suggest here.  I can't find any obvious source of information from NCBI on how they came to this set of Biotypes.  Certainly we can add more if/when we need to if we build parsers for Ensembl/Gencode GFFs.

Reply:
Have added a comment as suggested.